spec.html

```diff
@@ -30562,17 +30562,17 @@
         <h1>Array.prototype [ @@unscopables ]</h1>
         <p>The initial value of the @@unscopables data property is an object created by the following steps:</p>
         <emu-alg>
-          1. Let _blackList_ be ObjectCreate(*null*).
-          1. Perform CreateDataProperty(_blackList_, `"copyWithin"`, *true*).
-          1. Perform CreateDataProperty(_blackList_, `"entries"`, *true*).
-          1. Perform CreateDataProperty(_blackList_, `"fill"`, *true*).
-          1. Perform CreateDataProperty(_blackList_, `"find"`, *true*).
-          1. Perform CreateDataProperty(_blackList_, `"findIndex"`, *true*).
-          1. Perform CreateDataProperty(_blackList_, `"includes"`, *true*).
-          1. Perform CreateDataProperty(_blackList_, `"keys"`, *true*).
-          1. Perform CreateDataProperty(_blackList_, `"values"`, *true*).
+          1. Let _blocklist_ be ObjectCreate(*null*).
```

Comment:
This name makes no sense in this context. Nothing is being "blocked". If it must be changed, how about `unacceptableNameList`? Let's try to be intelligent about this, not mindless renaming zombies.